pb-jelly-gen/codegen/codegen.py

```diff
@@ -1012,7 +1011,29 @@ def gen_msg(
                 "pub static ref %s_default: %s = %s::default();" % (name, name, name)
             )
 
-        with block(self, "impl ::pb_jelly::Message for " + name):
+        with block(self, "impl ::pb_jelly::Serialize for " + name):
```

Comment:
`Serialize` and `Deserialize` are special words in `serde` and I think it'll be confusing to reuse the same terminology here - since it's not really referring to the same thing (and it's similarish?)

Reply:
I feel like they're common enough verbs that it won't be too confusing to see them used in another crate. These also happen to be the same verbs used in the (majority) of the public methods in each of these traits. Any other verbs I can think of would just be more confusing IMO. Were you able to come up with good alternative names?